config/updates.jenkins.io-parent.yaml

```
@@ -0,0 +1,116 @@
+        paths:
+          - path: /
+            serviceNameSuffix: files
+          - path: /.*[.](deb|hpi|war|rpm|msi|pkg|sha256|md5sum|zip|gz|pdf|json|svg|sh|jpeg|ico|png|html)$
+            pathType: ImplementationSpecific
+    tls:
+      - secretName: updates-jenkins-io-tls
+        hosts:
+          - azure.updates.jenkins.io
+
+  resources:
+    limits:
+      cpu: 500m
+      memory: 1024Mi
+    requests:
+      cpu: 500m
+      memory: 1024Mi
+
+  repository:
+    name: updates-jenkins-io-binary
+    persistentVolumeClaim:
+      enabled: true
+      spec:
+        accessModes:
+          - ReadOnlyMany
+        storageClassName: azurefile-csi-premium
+        volumeName: updates-jenkins-io-binary
+        resources:
+          requests:
+            storage: 2Gi  # See file share size in https://github.com/jenkins-infra/azure/blob/main/updates.jenkins.io.tf
+    # As the storage account is independently declared and created elsewhere (jenkins-infra/azure/updates.jenkins.io.tf),
+    # we're adding the PV definition to rattach the PVC to the existing storage account without creating a new one
+    # and especially without deleting it when the cluster has to be recreated.
+    persistentVolume:
+      enabled: true
+      spec:
+        capacity:
+          storage: 2Gi  # See file share size in https://github.com/jenkins-infra/azure/blob/main/updates.jenkins.io.tf
+        storageClassName: azurefile-csi-premium
+        accessModes:
+          - ReadOnlyMany
+        persistentVolumeReclaimPolicy: Retain
+        csi:
+          driver: file.csi.azure.com
+          readOnly: false
+          volumeHandle: updates-jenkins-io-binary  # make sure this volumeid is unique for every identical share in the cluster
+          volumeAttributes:
+            resourceGroup: updates-jenkins-io
+            shareName: updates-jenkins-io
+          nodeStageSecretRef:
+            name: updates-jenkins-io-mirrorbits-lite-binary
+            namespace: updates-jenkins-io
+        mountOptions:
+          - dir_mode=0755
+          - file_mode=0644
+          - uid=1000
+          - gid=1000
+          - mfsymlinks
+          - nobrl
+          - serverino
+          - cache=strict
+
+  nodeSelector:
+    agentpool: x86medium
+
+httpd:
+  enabled: true
+
+  resources:
+    limits:
+      cpu: 2000m
+      memory: 2048Mi
+    requests:
+      cpu: 2000m
+      memory: 2048Mi
+
+  repository:
+    name: updates-jenkins-io-binary
+    reuseExistingPersistentVolumeClaim: true
+
+  nodeSelector:
+    agentpool: x86medium
+
+rsyncd:
+  enabled: true
+
+  configuration:
+    components:
+      - name: jenkins
+        path: /rsyncd/data/jenkins
+        comment: "Jenkins Read-Only Mirror"
+        volume:
+          persistentVolumeClaim:
+            claimName: updates-jenkins-io-binary
+
+  resources:
+    limits:
+      cpu: 100m
+      memory: 128Mi
+    requests:
+      cpu: 50m
+      memory: 64Mi
+
+  nodeSelector:
+    agentpool: x86medium
```

```suggestion
rsyncd:
  enabled: true

  configuration:
    components:
      - name: jenkins
        path: /rsyncd/data/jenkins
        comment: "Jenkins Read-Only Mirror"
        volume:
          persistentVolumeClaim:
            claimName: updates-jenkins-io-binary

  resources:
    limits:
      cpu: 100m
      memory: 128Mi
    requests:
      cpu: 100m
      memory: 128Mi

  podSecurityContext:
    runAsUser: 65534 # User 'nobody'
    runAsGroup: 65534 # Group 'nogroup'
    runAsNonRoot: true

  containerSecurityContext:
    readOnlyRootFilesystem: true
    allowPrivilegeEscalation: false
    capabilities:
      drop:
      - ALL

  nodeSelector:
    agentpool: x86medium

```

- Requires https://github.com/jenkins-infra/helm-charts/pull/689, https://github.com/jenkins-infra/helm-charts/pull/688 and https://github.com/jenkins-infra/helm-charts/pull/687 to be merged (and the release 1.2.0 deployed, and the lock file updated here)
- Set up same rexsource requests and limits
- Harden execution by setting security contexts